docs/src/gui/qtvcp-development.adoc

```diff
@@ -171,10 +171,10 @@ After this function is called the widget has access to some special
 variables:
 
 *`self.HAL_GCOMP`*:: the _HAL component_ instance
-*`self.HAL_NAME`*:: This _widget's name_ as a string
-*`self.QT_OBJECT_`*:: This _widget's PyQt object instance_
-*`self.QTVCP_INSTANCE_`*:: The _very top level parent_ of the screen
-*`self.PATHS_`*:: The _instance of QtVCP's path_ library
+*`self.HAL_NAME`*:: this _widget's name_ as a string
+*`self.QT_OBJECT_`*:: this _widget's PyQt object instance_
+*`self.QTVCP_INSTANCE_`*:: the _very top level parent_ of the screen
+*`self.PATHS_`*:: the _instance of QtVCP's path_ library
 *`self.PREFS_`*:: the _instance of an optional preference file_
 *`self.SETTINGS_`*:: the `Qsettings` _object_
```

Comment:
Why not all beginning with capital T?

Reply:
The article is not needed in the first place IMHO. Did not want to do too much in that patch, I think.

Reply:
Shall I rebase and remove the articles at that opportunity?

Reply:
Look at https://linuxcnc.org/docs/2.9/html/gui/qtvcp-development.html#_widget_special_setup_functions
and decide which looks better to you - you'll find both "This" and "this" mixed in this section.
But it's more used with capital letter like in "4. Path Information"

Reply:
Shall I then rebase and change to capitals? I truly do not care either way.

Reply:
Yes :+1:

Reply:
done